13/umbraco-ui-builder/collections/retrieve-child-collections.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Configuring **one-to-many** relationships in Umbraco UI Builder, the backoffice UI builder for Umbraco.
+---
+
+# Retrieve Child Collections
+
+In a **one-to-many** relationship context, an entity from a parent collection can be associated with multiple entities from a child collection. In this context, you might need 
+to retrieve the child collection entities, without fetching the details of the parent.
+
+## Models Representation
+
+The Poco models would look like this:
```

Comment:
```suggestion
The models would look like this:
```

"Poco" probably isn't necessary, and would need defining.

Reply:
Updated